07/reshape-a-matrix.checkpoint.md

```diff
@@ -250,7 +333,7 @@ def reshape_matrix(matrix, r, c):
 
 ```py
 import unittest
-import main as p
+from main import *
 import numpy as np
```

Comment:
saw this mentioned in another review, that if we aren't using numpy, we should be able to remove this
```suggestion
```